packages/react-magma-dom/src/components/Stepper/Step.tsx

```diff
@@ -0,0 +1,242 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { CheckIcon, CrossIcon } from 'react-magma-icons';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { transparentize } from 'polished';
+import { BreakPointStyle } from './Stepper';
+import { HiddenLabelText } from '../Checkbox';
+
+/**
+ * @children required
+ */
+export interface StepProps extends React.HTMLAttributes<HTMLDivElement> {
+  breakpointStyle?: BreakPointStyle;
+  /*
+   * Index set by Stepper component
+   */
+  currentStep?: number;
+  /**
+   * Error state for each step
+   * @default false
+   */
+  hasError?: boolean;
```

Comment:
If only the current step can have an error, we should have the Stepper (parent) have the hasError prop. (@orion-cengage can you confirm if we expect a past step to have an error?)

Reply:
After speaking with Orion, it sounds like it's best we keep hasError at the step level.

Reply:
Discussed solution: We are going to allow any step to have an error state, not just the current step. Hopefully this will be an edge case.